torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -911,6 +915,55 @@ Tensor* TensorExprKernel::computeValue(const torch::jit::Value* v) {
           /*promote_inputs*/ false);
     }
 
+    case aten::nan_to_num: {
+      auto node = v->node();
+      return computeFourOperand(
+          "nan_to_num",
+          v,
+          [node](
+              const ExprHandle& input,
+              const ExprHandle& nan,
+              const ExprHandle& pos_inf,
+              const ExprHandle& neg_inf) {
```

Comment:
I'd be somewhat inclined to name these things like `nan_replacement` (or something).  I was confused seeing `nan` and `pos_inf` being used as default values and thought I'd found a bug when in fact I just demonstrated my lack of reading comprehension :-).

Anyways up to you.  I know the PT API docs use these names so, whatever :)

Reply:
yea, agreed that makes it easier to follow